data/schema/gui/widget_instances.cfg

```diff
@@ -43,6 +43,17 @@
         {DEFAULT_KEY "return_value" int 0}
         {DEFAULT_KEY "return_value_id" string ""}
     [/tag]
+    [tag]
+        name="combobox"
+        min="0"
+        max="infinite"
+        super="$generic/widget_instance"
+        {DEFAULT_KEY "history" string ""}
+        {DEFAULT_KEY "max_input_length" int 0}
+        {DEFAULT_KEY "label" t_string ""}
+        {DEFAULT_KEY "hint_text" t_string ""}
+        {DEFAULT_KEY "hint_image" string ""}
+    [/tag]
```

Comment:
It feels like something's missing here. Shouldn't there be a way to specify the preset list of options?

Reply:
Added in 29adbf8a3f06325bc64b546be03c381c12c67065.

Reply:
The `[option] `tag was missing.